inc/taskview.class.php

```diff
@@ -284,6 +284,20 @@ public function showJobLogs()
       });");
     }
 
+    /**
+    * Prepare data before update in database
+    *
+    * @param array $input
+    * @return array
+    */
+    public function prepareInputForUpdate($input)
+    {
+        if ($input['is_active'] && $input['is_active_from_db']) {
```

Comment:
1. Should rely on value from DB, it is safer than a values passed via a from.
2. `is_active` may be missing from input, for instance if task is updated via API.

```suggestion
        if ($this->fields['is_active'] && ($input['is_active'] ?? '1')) {
```